src/urllib3/connectionpool.py

```diff
@@ -65,6 +65,9 @@ class ConnectionPool(object):
     """
     Base class for all connection pools, such as
     :class:`.HTTPConnectionPool` and :class:`.HTTPSConnectionPool`.
+
+    .. note::
```

Comment:
Can we change the message to "ConnectionPool.urlopen() does not normalize or percent-encode target URIs which is useful if your target server doesn't support percent-encoded target URIs". What do you think?

Reply:
Seems good :)